Buildings/Resources/src/ThermalZones/Detailed/Validation/BESTEST/simulateAndPlot.py

```diff
@@ -412,6 +454,219 @@ def _find_data_bin(dataSet):
         results[i] = '{}'.format(int(counter))
     return results
 
+def export_data(data_set):
+    KWh = 3600*1e3
+    results = list()
+    for k in range(len(data_set)):
+        data = data_set[k]
+        if 'FF' in data['case']:
+            temp = {'case': data['case']}
+            for varVal in data['result']:
+                if varVal['variable'] == 'TRooHou.y':
+                    annual = varVal['value']
+                    Feb1 = annual[745:769]
+                    Jul14 = annual[4657:4681]
```

Comment:
Is this typo of code robust, e.g., does the `745:769` change if Dymola (or Optimica?) writes more or fewer output lines, perhaps when a future version changed the solver or has more events?
Perhaps you should first search at what line Feb 1 starts and ends, and then use this index.

Reply:
The function `def _extract_data(matFile, relVal):` from line 205 to line 243 converted the data point to (0~8760), so the start and end indices can be fixed here and it is flexible to handle more or less output lines.

```
# --------------------------------------------------------------------------------------------------
# Extract needed time series data from modelica simulation results
# --------------------------------------------------------------------------------------------------
def _extract_data(matFile, relVal):
    ...
```